### ISSUE TITLE
Expunge large objects from git repo, or migrate to toolkit repo

### ISSUE THREAD

**Issue by jchodera** [open-forcefield-group/open-forcefield-tools#27]
This repo contains some very large (now deleted) objects, bringing the repo to ~2GB. We should either [expunge those](https://help.github.com/articles/removing-files-from-a-repository-s-history/) or migrate this code to the main toolkit repo: https://github.com/openforcefield/openforcefield

**Comment by jchodera** [comment]
Ah. The problem is with `single-molecule-property-generation/`, which somehow ballooned to 1.5GB:
```
lski2414:open-forcefield-tools choderaj$ du -sh single-molecule-property-generation/
1.5G	single-molecule-property-generation/
```
@bmanubay and @mrshirts : Was this you?

**Comment by jchodera** [comment]
We shouldn't be checking datafiles into this repo. And now that we have, it's going to take some work to undo that.

I'm thinking we should just move the property calculator to the main toolkit and abandon this repo instead.

**Comment by bmanubay** [comment]
Yes, this was me. Sorry. I agree that this repo can be abandoned.
